### PR TITLE
Update: Docs for release

### PR DESCRIPTION
**Pull request by RaulTrombin**
Please check:
https://github.com/RaulTrombin/ping-rs/tree/Update_docs

### CODE REVIEW

**Review by ES-Alexander — CHANGES_REQUESTED** [review]
Some suggested content changes :-)

The main fundamental difference is emphasising that the library implements a protocol for device types, not specific devices - i.e. there can be multiple different devices that fit within / make use of the message set for a particular device type (e.g. Ping and Ping2 are both `Ping1D` type devices).

**Comment by RaulTrombin** [comment]
> Some suggested content changes :-)
> 
> The main fundamental difference is emphasising that the library implements a protocol for device types, not specific devices - i.e. there can be multiple different devices that fit within / make use of the message set for a particular device type (e.g. Ping and Ping2 are both `Ping1D` type devices).

Thks @ES-Alexander, I really appreciate the suggestions. Now it's easier to navigate the documentation and their related devices.

File: README.md [hunk 1]
```diff
@@ -1,16 +1,53 @@
-# ping-rs
-https://docs.bluerobotics.com/ping-rs/bluerobotics_ping/
+# :crab: Ping Library :sound:
 
-## Using example:
+[SONARs](https://en.wikipedia.org/wiki/Sonar), or **So**und **N**avigation **A**nd **R**anging devices, transmit sound waves and measure their reflections to detect surrounding obstacles and objects.
```
Comment (patrickelectric) [inline README.md:3]: That would probably be [SoNARs]

Comment (RaulTrombin) [reply README.md:3]: https://bluerobotics.com/learn/understanding-and-using-scanning-sonars/#operating-principles
What about replace `or`
SONARs, `from` Sound...

Comment (patrickelectric) [reply README.md:3]: Usually only the first letter is capitalized. But no hard feelings. Once @ES-Alexander approves it we can merge.

**Review by ES-Alexander — APPROVED** [review]

**Review by patrickelectric — APPROVED** [review]

**Pull request merged**